### PR TITLE
Created Dashboard skeleton

### PR DESCRIPTION
**Pull request by kqpham**
### What this PR does (required):
- Adds Dashboard skeleton to application
- Adds FE routes to necessary pages within Dashboard
- Added shared navbar between dashboard pages

### Screenshots / Videos (required):
- Post at least one screenshot to show the feature in action

https://user-images.githubusercontent.com/17209997/120558012-d0472400-c3cc-11eb-8ffb-9d0c64f68463.mov

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

- Logged in as a existing user, clicked on one of the navbar buttons to display the currently accessible dashboard pages.
- tested routing from dropdown menu within navbar to display profile page within dashboard.

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: client/src/components/AuthMenu/AuthMenu.tsx [hunk 1]
```diff
@@ -40,6 +45,9 @@ const AuthMenu = (): JSX.Element => {
         }}
         getContentAnchorEl={null}
       >
+        <MenuItem>
+          <Link to={to}>Profile</Link>
```
Comment (dproc96) [inline client/src/components/AuthMenu/AuthMenu.tsx:49]: Can you change this link to not be underlined and blue?

File: client/src/pages/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -4,13 +4,20 @@ import CircularProgress from '@material-ui/core/CircularProgress';
 import useStyles from './useStyles';
 import { useAuth } from '../../context/useAuthContext';
 import { useSocket } from '../../context/useSocketContext';
-import { useHistory } from 'react-router-dom';
+import { useHistory, Switch, Route } from 'react-router-dom';
 import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
 import { useEffect } from 'react';
+import { Box, Typography } from '@material-ui/core';
```
Comment (dproc96) [inline client/src/pages/Dashboard/Dashboard.tsx:10]: These two imports are never used

**Review by dproc96 — APPROVED** [review]
Two small change requests but if you make those, this is good to merge

File: client/src/components/AuthMenu/AuthMenu.tsx (hunk 1 above)
Comment (kqpham) [reply client/src/components/AuthMenu/AuthMenu.tsx:49]: Yes, I will get that done right away.

File: client/src/pages/Dashboard/Dashboard.tsx (hunk 2 above)
Comment (kqpham) [reply client/src/pages/Dashboard/Dashboard.tsx:10]: I have fixed this in my next pull request.

**Pull request merged**